how-to-guides/ci/run-github-actions-in-a-devbox.md

```diff
@@ -46,16 +59,16 @@ curl \
     -X POST \
     -H "Accept: application/vnd.github+json" \
     -H "Authorization: Bearer "${{ GITHUB_PAT }}"" \
-    https://api.github.com/repos/OWNER/REPO/actions/runners/registration-token
+    https://api.github.com/repos/ORGANIZATION/REPO/actions/runners/registration-token
 ```
 
 {% endhint %}
 
-### Configure and Start
+### Install and Start
 
 ```
-# Installs the actions service to run in background using systemd
-sudo ./svc.sh install && sudo ./svc.sh start
+# Installs the actions service to run in background using systemd and runs as root user
+sudo ./svc.sh install root && sudo ./svc.sh start
```

Comment:
run as root will simplify some issues as _sometimes_ things can be written as root and then can't be cleaned up 🙄 , TBH this seems like more of a github actions issue than us, but easiest this way